SPDY/SPDYStreamManager.m

```diff
@@ -21,8 +21,8 @@ @interface SPDYStreamNode : NSObject
 @implementation SPDYStreamNode
 {
   @public
-    __strong SPDYStreamNode *next;
-    __strong SPDYStreamNode *prev;
+    __weak SPDYStreamNode *next;
+    __weak SPDYStreamNode *prev;
```

Comment:
Your suggestion seems like a good pattern even though it isn't strictly necessary for this data structure.